fixtures/setup.json

```diff
@@ -1,23 +1,35 @@
 {
   "seed": 4,
   "keysToGen": 5,
+  "importKeys": [{"privateKey":"/8UoXvzGIU2H/TgTAmswx96m2dVCXkRqnIIlDhJrGCo=","sigType":2}],
   "preAllocGenKeys": [
     "10000000000",
     "10000000000",
     "10000000000",
     "10000000000",
+    "10000000000",
     "10000000000"
   ],
   "miners": [{
-    "owner": 0,
+    "owner": 5,
+    "sectorSize": 2048,
     "committedSectors": [{
-      "commR": {"/":"bafy2bzaceb4n26dwrtbzx7l6ifj32xmxkl5lc6lfzq4qlcoft3t2jzzs7anfi"},
-      "commD": {"/":"bafy2bzacec3p63bwba2ph3stn27jbiyadowkpdmtg2m7aysfciuevzrjpyndo"},
-      "sectorNum": 0,
+      "commR": {"/":"bafk4ehzaqugph7z7yajugw6wz6nn7x3akr3e32wecwexllfgqndnjt3nsaya"},
+      "commD": {"/":"bafk4chzavlx2s3zsomp6beohyevpbql477qyr7qnvgkvfwbbcqzfijo62arq"},
+      "sectorNum": 3,
```

Comment:
This is a bit weird, I'd expect numbers starting from zero. SectorNum is per-miner. 

Since we can't put documentation in the JSON files, could you please add a README for the fixtures that explains how this set-up all works? And how to re-generate it.

Reply:
Agreed. The presealed data is starts at 3 for this miner, so this is required. I'll add the README.

Reply:
Does the sector num get mixed in during sealing?  Does anything break if we just change this number here?  Keep in mind genesis sectors don't require verification of a PoRep they just get set in the miner state.

Reply:
Things do break if we change this number. First of all the sector number is in the file name of all the pre-sealed sectors on disk. We could rename those, but I think I tried that and something still broke.